### ISSUE TITLE
time tracking

### ISSUE THREAD

**Issue by ndvanforeest** [madisonmay/tracker#4]
HI,

In my clone of tracker I added a time stamp to the line that prints the values of the tracked attributes. This might be useful to others to, shall I add my addition as a pull request?

**Comment by madisonmay** [comment]
Definitely, would love your contribution. Happy to look over your PR and merge things in.

**Comment by ndvanforeest** [comment]
I'll send it somewhere today.

I really like that you used the word "pesky" in tracker. I read about
tracker via python weekly, but forgot the name (i.e., tracker). But a
search on "pesky github" on google was a hit in one :-)
